packages/product-react/src/components/Container/Container.scss

```diff
@@ -0,0 +1,78 @@
+$lucia-container-margin-1: 0 12px;
```

Comment:
now that we're distributing new styles, this needs to be part of the build.

we need to output the compiled scss as part of `dist`